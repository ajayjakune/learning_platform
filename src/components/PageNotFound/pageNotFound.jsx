import React from 'react'
import { Col, Row, Spinner, Table } from 'react-bootstrap'

export default function PageNotFound() {
    return (
        <div className="row justify-content-center jumbotron">
            <table>
                <th></th>
                <th><Spinner animation="grow" variant="danger" size="sm" />
                    <Spinner animation="grow" variant="danger" />
                </th>
                <tr>
                    <td>
                        <h1>Error 404!!! Page Not Found</h1>
                    </td>
                    <td></td>
                </tr>
            </table>
        </div>
    )
}
