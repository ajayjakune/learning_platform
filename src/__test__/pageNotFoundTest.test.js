import React from 'react';
import ReactDom from 'react-dom';
import { render, cleanup } from '@testing-library/react'
import PageNotFound from '../components/PageNotFound/pageNotFound';

afterEach(cleanup)

it("renders without crashing", () => {
    const { asFragment } = render(<PageNotFound />)

    expect(asFragment(<PageNotFound />)).toMatchSnapshot()
})

