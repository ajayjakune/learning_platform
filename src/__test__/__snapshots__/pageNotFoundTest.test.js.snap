// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="row justify-content-center jumbotron"
  >
    <table>
      <tbody>
        <tr>
          <th />
          <th>
            <div
              class="spinner-grow spinner-grow-sm text-danger"
            />
            <div
              class="spinner-grow text-danger"
            />
          </th>
        </tr>
        <tr>
          <td>
            <h1>
              Error 404!!! Page Not Found
            </h1>
          </td>
          <td />
        </tr>
      </tbody>
    </table>
  </div>
</DocumentFragment>
`;
